packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,232 @@
+part of firebase_remote_config;
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig extends ChangeNotifier {
+  static const MethodChannel _channel =
```

Comment:
The other plugins make this public and @visibleForTesting to avoid duplicating the channel name in the test, perhaps we should do the same here?

Reply:
Done